site/templates/message.html

```diff
@@ -53,122 +56,134 @@ <h2>Send Brian a message on his Tidbyt.</h2>
           </select>
-          <div class="form-text"><a href="https://github.com/tidbyt/pixlet/blob/main/docs/fonts.md">Font Examples</a>
+          <div class="form-text">
+            <a href="https://github.com/tidbyt/pixlet/blob/main/docs/fonts.md">Font Examples</a>
           </div>
         </div>
 
         <div class="row justify-content-center">
           <div class="col-auto">
             <button id="submit-button" type="submit" class="btn btn-primary">
-              <span id="submit-button-span" style="display: none" class="spinner-grow spinner-grow-sm" role="status"
-                aria-hidden="true"></span>
-              Send Message
+              <span class="spinner-grow spinner-grow-sm" role="status" aria-hidden="true"></span>
+              <span class="text">Send Message</span>
             </button>
           </div>
         </div>
 
         <div class="row justify-content-center">
           <div class="col-auto" id="message-response-div"></div>
         </div>
-
       </form>
     </div>
   </div>
 </div>
 
 <div class="container">
   <div class="row justify-content-center" id="stop_button">
-
     <div class="col-auto">
       <h2>Other Tidbyt actions:</h2>
     </div>
 
     <div class="col-auto">
       <button onclick="sendStop()" type="button" class="btn btn-danger" id="stop-button">
-        <span id="stop-button-span" style="display: none" class="spinner-grow spinner-grow-sm" role="status"
-          aria-hidden="true"></span>
-        Tell Brian to stop talking.
+        <span class="spinner-grow spinner-grow-sm" role="status" aria-hidden="true"></span>
+        <span class="text">Tell Brian to stop talking.</span>
       </button>
     </div>
-
   </div>
 
-  <div class="row justify-content-center" id="response_messages"></div>
+  <div class="row justify-content-center" id="stop-response-div"></div>
 </div>
 
 <script>
-  function removeAllChildNodes(parent) {
-    while (parent.firstChild) {
-      parent.removeChild(parent.firstChild);
-    }
-  }
-
-  // toggle button
-  async function toggleButton(buttonId, spinnerSpanId, enabledText, disabledText) {
-    const btn = document.getElementById(buttonId);
-    const btnSpan = document.getElementById(spinnerSpanId);
-
-    const btnContent = btn.textContent;
-
-    if (btnContent.includes(enabledText)) {
-      btn.setAttribute("disabled", "");
-      btnSpan.removeAttribute("style");
-      const inner = btn.innerHTML;
-      const newInner = inner.replace(enabledText, disabledText);
-      btn.innerHTML = newInner;
-    } else if (btnContent.includes(disabledText)) {
-      btn.removeAttribute("disabled");
-      btnSpan.setAttribute("style", "display: none");
-      const inner = btn.innerHTML;
-      const newInner = inner.replace(disabledText, enabledText);
-      btn.innerHTML = newInner;
-    }
-  }
+  const IS_MOCKING = true;
+  const mockDelay = (seconds) => new Promise((resolve) => setTimeout(resolve, seconds * 1000));
```

Comment:
You can remove the mocking stuff or change it to false, but I added this so I could test the API without actually triggering it.